src/core/particle_data.cpp

```diff
@@ -2304,8 +2295,3 @@ bool particle_exists(int part) {
         return true;
    return false;
 }
-
-#ifndef MASS
-constexpr double ParticleProperties::mass;
```

Comment:
Okay, I didn't know that and the tests succeeded, can you explain why this is needed?